ui/app/pages/swaps/swaps-gas-customization-modal/swaps-gas-customization-modal.container.js

```diff
@@ -121,18 +128,20 @@ const mapStateToProps = (state) => {
     customGasLimitMessage,
     customTotalSupplement,
     usdConversionRate: getUSDConversionRate(state),
-    disableSave: insufficientBalance || customGasLimit < minimumGasLimit,
+    disableSave: insufficientBalance || customGasLimitTooLow,
     minimumGasLimit,
   }
 }
 
 const mapDispatchToProps = (dispatch) => {
   return {
     cancelAndClose: () => {
+      dispatch(swapCustomGasModalClosed())
```

Comment:
Ensure that custom modal gas limit and price amounts are reset when closing the modal